src/main/java/com/gmail/goosius/siegewar/enums/SiegeStatus.java

```diff
@@ -72,6 +72,35 @@ public String getName() {
 		return this.name;
 	}
 
+	public boolean allowsPlundering() {
+		return attackersWon();
```

Comment:
🚧 
This is not quite right. The SiegeType also has to be conquest to allow plunder.